libcloudforensics/providers/gcp/forensics.py

```diff
@@ -69,13 +72,13 @@ def CreateDiskCopy(
       instance = src_project.compute.GetInstance(instance_name)
       disk_to_copy = instance.GetBootDisk()
 
-    common.LOGGER.info('Disk copy of {0:s} started...'.format(
+    logger.info('Disk copy of {0:s} started...'.format(
         disk_to_copy.name))
     snapshot = disk_to_copy.Snapshot()
     new_disk = dst_project.compute.CreateDiskFromSnapshot(
         snapshot, disk_name_prefix='evidence', disk_type=disk_type)
     snapshot.Delete()
```

Comment:
You can also add the intermediary steps here (snapshot created, disk created from snapshot, snapshot deleted), maybe as `.debug()` statements if you want to be able to use the log level to chose what to display and what not.